From: Colin Ian King <colin.king@canonical.com>
Date: Tue, 23 Jun 2020 09:42:07 +0100
Subject: libbpf: Fix spelling mistake "kallasyms" -> "kallsyms"
Patch-mainline: v5.9-rc1
Git-commit: 135c783f4794fbdeace4a969dea6eabd27f8a501
References: bsc#1177028

There is a spelling mistake in a pr_warn message. Fix it.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Acked-by: Andrii Nakryiko <andriin@fb.com>
Link: https://lore.kernel.org/bpf/20200623084207.149253-1-colin.king@canonical.com
Acked-by: Gary Lin <glin@suse.com>
---
 tools/lib/bpf/libbpf.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/lib/bpf/libbpf.c
+++ b/tools/lib/bpf/libbpf.c
@@ -5749,7 +5749,7 @@ static int bpf_object__read_kallsyms_fil
 		if (ret == EOF && feof(f))
 			break;
 		if (ret != 3) {
-			pr_warning("failed to read kallasyms entry: %d\n", ret);
+			pr_warning("failed to read kallsyms entry: %d\n", ret);
 			err = -EINVAL;
 			goto out;
 		}
